### PR TITLE
CFC 2 Conclusion Fix

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Podium
- Fix Table Formatting
- Fix Tournament Name (Request by the Organiser)
- Add Mapper name on the Mappools
- Some escaping

---

### CODE REVIEW

File: wiki/Tournaments/CFC/2/en.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
+---
+tags:
+- CFC 2
+- CFC2
+---
+# Catch French Cup 2
+
+![CFC 2 Logo](img/logo.png)
+
+The **Catch French Cup 2** (***CFC 2***) was a one-on-one French osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It was the second installment of the Catch French Cup.
```
Comment (Joehuu) [inline wiki/Tournaments/CFC/2/en.md]: `second` -> `2nd`

File: wiki/Tournaments/CFC/2/en.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+| Referees | ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [Ryumi](/users/6596270), ![][flag_FR] [Rsuiono](/users/9266902), ![][flag_FR] [TLQ_Yoshii](/users/7157133), ![][flag_FR] [300BPM JUMPER](/users/1893667), ![][flag_FR] [Purettsu Eru](/users/1542565) |
+| Streamers | ![][flag_FR] [Purettsu Eru](/users/1542565), ![][flag_FR] [FulgurentKille](/users/4096322), ![][flag_FR] [300BPM JUMPER](/users/1893667), ![][flag_FR] [Kaeldori](/users/962519), ![][flag_FR] [Star Sapphire](/users/2953898) | 
+| Commentators | ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [FulgurentKille](/users/4096322), ![][flag_FR] [Corn-](/users/4089232), ![][flag_FR] [Orazeit](/users/8335913), ![][flag_FR] [Star Sapphire](/users/2953898), ![][flag_FR] [4L-Chan 3 Porte](/users/7253987) |
+| Wiki | ![][flag_FR] [Kasumii-sama](/users/6177263) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/714536)
+- [Livestream](https://www.twitch.tv/osufrlive)
+- **[Main Spreadsheet](https://docs.google.com/spreadsheets/d/1AwvOj_yBpcrB5sLEtRpEE7Wq7ZTVSZRncKzPjs7T-xQ/edit?usp=sharing)**
```
Comment (Joehuu) [inline wiki/Tournaments/CFC/2/en.md]: `Main Spreadsheet` -> `Statistics sheet`

File: wiki/Tournaments/en.md [hunk 3]
```diff
@@ -120,7 +120,8 @@ Unofficial tournaments/competitions hosted by the communities.
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
 | --- | --- | --- | --- | --- | --- |
 | Coupe de France CTB | 2017-10-28 | 2017-12-16 | ![][flag_FR] [Noulayfe](/users/4316542) | ![][flag_FR] [Alostis](/users/2986875) | ![][flag_FR] [Le grand casque](https://osu.ppy.sh/users/5490623) |
-| [Catch French Cup 2018](/wiki/Tournaments/CFC/2018/) | 2018-04-07 | 2018-05-06 | ![][flag_FR] [Le grand casque](https://osu.ppy.sh/users/5490623) | ![][flag_FR] [Julo](https://osu.ppy.sh/users/4203239) | ![][flag_FR] [Electr0o](https://osu.ppy.sh/users/9484428) |
+| [Catch French Cup 2](/wiki/Tournaments/CFC/2) | 2018-04-07 | 2018-05-06 | ![][flag_FR] [Le grand casque](https://osu.ppy.sh/users/5490623) | ![][flag_FR] [Julo](https://osu.ppy.sh/users/4203239) | ![][flag_FR] [Electr0o](https://osu.ppy.sh/users/9484428) |
+
```
Comment (TPGPL) [inline wiki/Tournaments/en.md]: delete this line

File: wiki/Tournaments/CFC/2/en.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![][flag_FR] [Le grand casque](/users/5490623) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_FR] [Julo](/users/4203239) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_FR] [Electr0o](/users/9484428) |
+
+
```
Comment (Joehuu) [inline wiki/Tournaments/CFC/2/en.md]: Remove this extra line.

**Pull request merged**